README.md

```diff
@@ -118,7 +124,115 @@ things like new user registration, and those settings are also configured within
  `politeiawww.conf`. The current code should work with most SSL-based SMTP servers
 (but not TLS) using username and password as authentication.
 
-#### 4. Build the programs:
+#### 4. Setup Cockroachdb:
+
+politeiad stores proposal data in git repositories that are regularly backed up
+to github and are cryptographically timestamped onto the Decred blockchain at
+periodic intervals.  The politeiad git repositories serve as the source of
+truth for proposal data and a CockroachDB database acts as a cache for proposal
+data.
+
+politeiad has read and write access to the cache.  politeiawww has only read
+access to the cache.  The flow of data is as follows:
+
+1. politeiawww receives a command from a user
+2. politeiawww creates a politeiad request for the command and sends it
+3. politeiad writes the command data to the git repository and updtes the cache
+4. politeiad returns a response to politeiawww
+5. politeiawww reads the cache if needed
+6. politeiawww returns a response to the user
+
+Install CockroachDB using the instructions found in the [CockroachDB
+Documentation](https://www.cockroachlabs.com/docs/stable/install-cockroachdb-mac.html).
+
+Run the following commands to setup a local CockroachDB instance:
+
+```
+COCKROACHDB_DIR="/Users/<user>/Library/Application Support/cockroachdb"
```

Comment:
This probably should become a script that we link from doco.

Reply:
Fixed.